spec/src/main/asciidoc/fallback.asciidoc

```diff
@@ -83,8 +83,14 @@ This is used if the `fallbackMethod` is on the same class as the method to call
 
 The above code snippet means when the method failed and retry reaches its maximum retry, the fallback operation will be performed.
 The method `fallbackForServiceB` will be invoked.
-The return type of `fallbackForServiceB` must be `String` and the argument list for `fallbackForServiceB` must be the same as `ServiceB`.
-Otherwise, the `FaultToleranceDefinitionException` exception will be thrown.
+
+When `fallbackMethod` is used a `FaultToleranceDefinitionException` will be thrown if any of the following constraints are not met:
+
+* The named fallback method must be on the same class, a superclass or an implemented interface of the class which declares the annotated method
+* The named fallback method must have the same parameter types as the annotated method (after resolving any type variables)
+* The named fallback method must have the same return type as the annotated method (after resolving any type variables)
+* If the named fallback method is not declared by the same class as the annotated method, it must not be `private`
+* If the named fallback method is not in the same package as the annotated method, it must be either `protected` or `public`
```

Comment:
see above. Can be deleted with the above suggestion.

Reply:
Done